src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from "react-dom";
+import './index.css';
+import App from './App';
+import reportWebVitals from './reportWebVitals';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7256353)
```suggestion
import reportWebVitals from "./reportWebVitals";
```